manifests/pip.pp

```diff
@@ -221,14 +219,19 @@
 
       default: {
         # Anti-action, uninstall.
-        $pip_exec_name = "pip_uninstall_${name}"
+        $uninstall = true
```

Comment:
Do we need a extra variable? Can't we just use $ensure?

Reply:
No. Yes. I`ll implement that.